app/views/ProfileView/index.js

```diff
@@ -436,6 +436,7 @@ class ProfileView extends React.Component {
 		logEvent(events.PROFILE_LOGOUT_OTHER_LOCATIONS);
 		showConfirmationAlert({
 			message: I18n.t('You_will_be_logged_out_from_other_locations'),
+			confirmationText: I18n.t('OK'),
 			callToAction: I18n.t('Logout'),
```

Comment:
There was a PR merged right after `Logout from other locations feature` which refactored `showConfirmationAlert` renaming `callToAction` to `confirmationText`.

This might work without other changes:
```suggestion
			confirmationText: I18n.t('Logout'),
```

Reply:
Should I just close my PR then? I think that PR should solve the issue. I can send in another PR if the issue keeps occurring.

Reply:
I'll fix this.

Reply:
@diegolmello Do have a look when you're free. I've resolved it!